### ISSUE TITLE
Review resistors and attenuation in RF lines

### ISSUE THREAD

**Issue by rfrht** [rfrht/FT991A-PAT#14]
Resistors R5, R7 and R10 are essentially attenuators in RF lines - this was blindly copied from the original G4HUP project, with no test of what's the effect in gain output.

As per [G4HUP statement](http://icas.to/PAT/PAT%20V2%20description.pdf) in his project description:

    [...]and a resistive attenuator to allow the overall gain of the board to be matched to the input requirements of the SDR.

Will test in a VNA what's the effect of these transistors, especially the R5 which can cause an attenuation in the input signal.

**Comment by rfrht** [comment]
# TL;DR:
The resistors are indeed important and needed.

# Long-winded
The resistors are crucial for matching the loads and ensuring the best performance of the amplifiers. I experimented in an speculative/exploratory by entirely removing and changing the values with the resistors that I had around. Findings:

* The first `R5` 100K resistor ensures signal stability.
* The `R7` resistor already provides the best performance at 1K
* `R8` and `R10` had their values changed slightly; `R8` changed from 39K to 10K, `R10` from 100R to 220R.

That yielded a modest 0.3 dB gain increase.

# Original component values VNA sweep
![pat-estudo-inicial](https://user-images.githubusercontent.com/3044503/96876776-3b03aa80-144f-11eb-9b34-c64e62289027.png)


# New component values VNA sweep
![pat-estudo-final](https://user-images.githubusercontent.com/3044503/96876770-39d27d80-144f-11eb-89d3-93a0537a99d8.png)

**Comment by rfrht** [comment]
Included in c3bcb1ba.
